### PR TITLE
test: add cluster name support test scenario

### PR DESCRIPTION
**Pull request by leseb**
We need to test the cluster name support in this CI as well. This
commit might be prone to debate because it tests 2 things in a single
scenario. We first test our ability to deploy a cluster AND the cluster
name support. However it's easier to do it this way and will reduce the
amount of time for testingg. If we don't do this we will have a
duplicate those 2 existing tests into new ones 'only' to test the
cluster name support.

Signed-off-by: Sébastien Han <seb@redhat.com>

### CODE REVIEW

**Comment by leseb** [comment]
Excepted failure from the tests which are looking for a cluster named 'ceph'.

**Comment by leseb** [comment]
Updated the tests with the new cluster name.

**Comment by leseb** [comment]
jenkins test this please

**Comment by leseb** [comment]
jenkins test ansible2.2-dmcrypt_journal

**Comment by leseb** [comment]
I think we have an issue with the ceph repos, experiencing the same problem on my laptop.
I'll re-run jenkins later tomorrow.

File: tests/scenarios/example.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
 # Basic information about ceph and its configuration
 ceph = {
     'releases': ['jewel', 'infernalis'],
-    'cluster_name': 'ceph'
+    'cluster_name': 'test'
```
Comment (leseb) [inline tests/scenarios/example.py]: @andrewschoen Is this one still needed?

Comment (andrewschoen) [inline tests/scenarios/example.py]: @leseb nope, that entire example could actually be removed. In https://github.com/ceph/ceph-ansible/pull/1150 we removed the need for a ``scenario.py`` file at all.

Comment (leseb) [inline tests/scenarios/example.py]: Alright, let me remove it then.

**Comment by andrewschoen** [comment]
jenkins test this please

**Comment by andrewschoen** [comment]
jenkins test ansible2.2-dmcrypt_journal

**Comment by andrewschoen** [comment]
jenkins test ansible2.2-docker_cluster

**Comment by andrewschoen** [comment]
jenkins test ansible2.2-dmcrypt_journal_collocation

**Comment by leseb** [comment]
jenkins test ansible2.2-dmcrypt_journal_collocation

**Comment by leseb** [comment]
jenkins test ansible2.2-dmcrypt_journal

**Comment by leseb** [comment]
jenkins test ansible2.2-dmcrypt_journal_collocation

**Comment by leseb** [comment]
@andrewschoen expected bug from the CI, this is related to this issue: https://github.com/ceph/ceph/pull/12033

**Comment by leseb** [comment]
@andrewschoen let me look at the docker error. Once it's fixed we can merge this, we just need to wait for the backport in Ceph to fix the dmcrypt stuff.

**Comment by andrewschoen** [comment]
@leseb ok, if the dymcrypt scenarios won't pass until that is backported then let's take ``cluster`` out of the group_vars for those scenarios and maybe add a note?

**Comment by leseb** [comment]
@andrewschoen done

**Comment by andrewschoen** [comment]
Once we have ``docker_cluster`` green this is good to go.

**Comment by leseb** [comment]
jenkins test ansible2.2-docker_cluster

**Comment by leseb** [comment]
jenkins test ansible2.2-docker_cluster

**Comment by leseb** [comment]
jenkins test ansible2.2-docker_cluster

**Comment by leseb** [comment]
@andrewschoen docker CI problem seems to be caused by the VM itself.

**Comment by andrewschoen** [comment]
@leseb on that last test, yes. but, we have successful runs of ``ansible2.2-docker_cluster`` in other PRs.

**Comment by andrewschoen** [comment]
jenkins test ansible2.2-docker_cluster

**Comment by andrewschoen** [comment]
@leseb this latest run seems like a real failure:

https://jenkins.ceph.com/job/ceph-ansible-prs-ansible2.2-docker_cluster/113/console

**Comment by andrewschoen** [comment]
jenkins test ansible2.2-docker_cluster

**Comment by andrewschoen** [comment]
It seems one of our jenkins slaves was having troubles, I've taken it offline for now.

https://jenkins.ceph.com/computer/172.21.9.33+ceph-builders/

**Comment by andrewschoen** [comment]
jenkins test ansible2.2-docker_cluster

**Comment by leseb** [comment]
jenkins test ansible2.2-centos7_cluster

**Comment by leseb** [comment]
jenkins test ansible2.2-docker_cluster

**Comment by leseb** [comment]
jenkins test ansible2.2-xenial_cluster

**Comment by andrewschoen** [comment]
jenkins test ansible2.2-docker_cluster

**Pull request merged**